src/index.html

```diff
@@ -7,6 +7,465 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__link header__link--logo">
+        <img
+          src="../src/images/logo.png"
+          alt="Moyo logotype"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link nav__link--active is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptopsAndComputers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <section class="catalog__section">
+        <div class="catalog__product-card">
+          <div
+            class="product-card"
+            data-qa="card"
+          >
+            <img
+              src="./images/imac.jpeg"
+              class="product-card__image"
+              alt ="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+          </img>
+            <article class="product-card__description">
```

Comment:
Do you need extra wrapper for the elements after the card image?
Will be better to get rid of it